scripts/test.sh

```diff
@@ -65,6 +65,7 @@ EXAMPLES:
 # Parse command-line arguments.
 RUN_UNIT_TESTS="true"
 RUN_INTEGRATION_TESTS="true"
+RUN_BENCHMARKS="true"
```

Comment:
It looks like this is left-over from an older version of this PR where the benchmarks were run as a part of the test suite. I think it probably makes sense to remove all changes to `./scripts/test.sh`.